content/en/docs/instrumentation/java/manual.md

```diff
@@ -46,7 +213,56 @@ telemetry source-identifying info.
             </dependency>
         </dependencies>
     </dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-api</artifactId>
+        </dependency>
+    </dependencies>
+</project>
+```
+
+{{% /tab %}}
+{{% /tabpane %}}
+
+<!-- prettier-ignore-end -->
+
+### Initialize the SDK
+
+{{% alert title="Note" color="info" %}} If you’re instrumenting a library,
+**skip this step**. {{% /alert %}}
+
+If you instrument a Java app, install the dependencies for the OpenTelemetry
+SDK.
+
+{{< tabpane text=true >}} {{% tab Gradle %}}
+
+```kotlin
+dependencies {
+    implementation 'io.opentelemetry:opentelemetry-api:{{% param javaVersion %}}'
+    implementation 'io.opentelemetry:opentelemetry-sdk:{{% param javaVersion %}}'
+    implementation 'io.opentelemetry:opentelemetry-sdk-metrics:{{% param javaVersion %}}'
```

Comment:
This dependency is not needed - its packaged with `opentelemetry-sdk`.

Reply:
Oddly, if I do not include `-metrics` the setup with auto-configurations throws me an error, maybe thats a bug?

```
***************************
APPLICATION FAILED TO START
***************************

Description:

An attempt was made to call a method that does not exist. The attempt was made from the following location:

    io.opentelemetry.sdk.autoconfigure.MeterProviderConfiguration.configureMeterProvider(MeterProviderConfiguration.java:47)

The following method did not exist:

    'io.opentelemetry.sdk.metrics.internal.exemplar.ExemplarFilter io.opentelemetry.sdk.metrics.internal.exemplar.ExemplarFilter.traceBased()'

The calling method's class, io.opentelemetry.sdk.autoconfigure.MeterProviderConfiguration, was loaded from the following location:

    jar:file:/Users/neumanns/Projects/manual-java/java-simple/build/libs/java-simple.jar!/BOOT-INF/lib/opentelemetry-sdk-extension-autoconfigure-1.28.0.jar!/io/opentelemetry/sdk/autoconfigure/MeterProviderConfiguration.class

The called method's class, io.opentelemetry.sdk.metrics.internal.exemplar.ExemplarFilter, is available from the following locations:

    jar:file:/Users/neumanns/Projects/manual-java/java-simple/build/libs/java-simple.jar!/BOOT-INF/lib/opentelemetry-sdk-metrics-1.19.0.jar!/io/opentelemetry/sdk/metrics/internal/exemplar/ExemplarFilter.class

The called method's class hierarchy was loaded from the following locations:

    io.opentelemetry.sdk.metrics.internal.exemplar.ExemplarFilter: jar:file:/Users/neumanns/Projects/manual-java/java-simple/build/libs/java-simple.jar!/BOOT-INF/lib/opentelemetry-sdk-metrics-1.19.0.jar!/


Action:

Correct the classpath of your application so that it contains compatible versions of the classes io.opentelemetry.sdk.autoconfigure.MeterProviderConfiguration and io.opentelemetry.sdk.metrics.internal.exemplar.ExemplarFilter
```

Reply:
Oh this is dependency collision issue with spring boot dependency management. I've come across this before. Just did a quick search and found this message I send about this:
>I’ve just discovered something rather annoying upgrading to spring boot 3. If you use the [io.spring.dependency-management](https://docs.spring.io/dependency-management-plugin/docs/current/reference/html/), it brings in dependencies in a way that conflicts and overrides our opentelemetry-bom. I.e. I was using opentelemetry-bom:1.22.0 and yet opentelemetry-api and other artifacts were resolving to 1.19.0. After some investigation I determined that spring boot has added opentelemetry artifacts at version 1.19.0 to its set of managed dependencies: https://docs.spring.io/spring-boot/docs/3.0.1/reference/htmlsingle/#appendix.dependency-versions
I got around it by bailing on the io.spring.dependency-management plugin and instead just depending on the spring boot BOM like this: [https://github.com/newrelic/newrelic-opentelemetry-examples/pull/258/commits/46571423558a46a49229617f5872c1135f8c9490#diff-21bc7265[…]7eb2c226b4b6b5](https://github.com/newrelic/newrelic-opentelemetry-examples/pull/258/commits/46571423558a46a49229617f5872c1135f8c9490#diff-21bc7265f74c0805ef365c54fd9e2696b565e660a8f7237ecd7eb2c226b4b6b5)

Notice how the error message is referencing `opentelemetry-sdk-metrics-1.19.0`? That's the sign that the opentelemetry-bom is being overridden.

Reply:
let me take a look ...